proto/orchestrator.proto

```diff
@@ -228,6 +228,7 @@ message GetCloudServiceRequest { string service_id = 1; }
 
 message RegisterCloudServiceRequest { CloudService service = 1; }
 
+// TODO(oxisto): Why separate service_id when it is in service already?
```

Comment:
Because Cloud service is only embedded in the request. the actual id belonging to the request is in service_id. This is also used in the HTTP path. Ref. to style guide ;)